src/intTest/resources/outbound_uat_data/acceptance/type1-birth-all-fields.fhir.json

```diff
@@ -0,0 +1,95 @@
+{
+  "parameter": [
+    {
+      "name": "gpTradingPartnerCode",
+      "valueString": "TES5"
+    },
+    {
+      "name": "acceptanceCode",
+      "valueString": "A"
+    },
+    {
+      "name": "acceptanceType",
+      "valueString": "1"
+    },
+    {
+      "name": "acceptanceDate",
+      "valueString": "1992-01-13"
+    },
+    {
+      "name": "freeText",
+      "valueString": "Free text value test abcd"
```

Comment:
Should use uppercase since we ask TP to do so.

Also I've merged in a branch with examples, might want to enable / replace / deduplicate as needed

Reply:
I'll check examples in next PR